.github/ISSUE_TEMPLATE/security.md

```diff
@@ -0,0 +1,16 @@
+---
+name: 👮 Security Issue
+about: Responsible Disclosure
+---
+
+## 👮 [Responsible Disclosure](https://github.com/nodejs/security-wg/blob/master/processes/responsible_disclosure_template.md)
+
+Do not open issues that might have security implications. It is critical that security related issues
+are reported privately so we have time to address them before they become public knowledge. Please
+notify all security related issues via the following channels:
+
+a) [node-security-wg](https://github.com/nodejs/security-wg)
+
+b) [hackerone](https://www.hackerone.com/)
```

Comment:
this is not correct. There is a more detailed process on reporting vulna defined in security-wg. The security-wg uses hackerone.

Reply:
Should I copy the process here or just redirect to security-wg to follow the steps there?

Reply:
Just copy https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md#process